modules/schematrons/pages/index.adoc

```diff
@@ -24,6 +25,47 @@ The individual Schematron files are executed in a logical sequence:
 . Check the values of leaf elements are consistent with each other.
 . For `dynamic`, check rules that depend on information outside of the notice being validated.
 
+=== Phases
+
+Starting from SDK version 1.10.0, the Schematron files make use of phases, a feature of the Schematron standard. A phase defines a named group of active patterns, and you can set the phase to use when executing the Schematron validation, thus defining the subset of rules to be executed.
+
+The various patterns in the schematron files contain either rules that apply only to a specific notice subtype, or rules that apply to any notice.
+
+We have defined one phase for each notice subtype, with an identifier equal to the notice subtype, and all the rules that must be applied to notices of that subtype: all patterns for this subtype, and all patterns not specific to a subtype.
+
+The definition of the phases is in the `complete-validation.sch` file:
+
+[source,xml]
+----
+<phase id="16"> <!--1-->
+    <active pattern="EFORMS-validation-stage-1a" /> <!--2-->
+    <active pattern="EFORMS-validation-stage-1b-16" /> <!--3-->
+    <active pattern="EFORMS-validation-stage-2a-16" />
+    <active pattern="EFORMS-validation-stage-2b" />
+    <active pattern="EFORMS-validation-stage-3a" />
+    <active pattern="EFORMS-validation-stage-3b" />
+    <active pattern="EFORMS-validation-stage-3b-16" />
+    <active pattern="EFORMS-validation-stage-4-16" />
+    <active pattern="EFORMS-validation-stage-5" />
+</phase>
+<phase id="17">
+    <active pattern="EFORMS-validation-stage-1a" />
+    <active pattern="EFORMS-validation-stage-1b-17" />
+    ...
+----
+<1> Definition of a phase with its identifier.
+<2> Pattern containing rules not specific to a subtype.
+<3> Pattern containing rules that apply only to notices with subtype 16.
+
```

Comment:
Do I understand correctly when I say that:

- the phase ID matches the notice subtype it applies to,
- notice subtype specific set of rules are identifiable with an additional " - _notice-subtype_ " appendix to the pattern?

Reply:
Yes, that is correct.
It's just a convention to have unique identifiers. People who use the Schematron don't need to know this, so I haven't documented it.